src/reporting/Market.sol

```diff
@@ -188,12 +188,16 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
     }
 
     function tryFinalize() public returns (bool) {
-        require(tentativeWinningPayoutDistributionHash != bytes32(0));
-
         if (getReportingState() != ReportingState.AWAITING_FINALIZATION) {
             return false;
         }
 
+        // If we're waiting for finalization and no reports have been recieved we move to the next reporting window
+        if (!hasReceivedAReport()) {
+            IReportingWindow _newReportingWindow = getBranch().getNextReportingWindow();
+            return migrateReportingWindow(_newReportingWindow);
+        }
```

Comment:
This code _feels_ like it should go along with the market migration stuff?  Though, I believe currently the market migration stuff is all related to forking, so perhaps not.

Reply:
I _believe_ that returning false from this method is meant to indicate that the market was not finalized and returning true means the market was finalized.  Based on that, I think this if block should _always_ return false (regardless of whether the market migrated successfully or not).

Reply:
Will do. And yea the current migration stuff is pretty fork specific.